src/cljam/io/bam/decoder.clj

```diff
@@ -124,14 +142,18 @@
          qname       (lsb/read-string buffer (dec l-read-name))
          _           (lsb/skip buffer 1)
          cigar-bytes (lsb/read-bytes buffer (* n-cigar-op 4))
-         [cigar ^long len] (cigar/decode-cigar-and-ref-length cigar-bytes)
+         [cigar ^long len first-cigar-operation-clips-entire-read-flag] (cigar/decode-cigar-and-ref-length cigar-bytes l-seq)
```

Comment:
Is `first-cigar-operation-clips-entire-read-flag` too long? I couldn't think of a good variable name. 
If you have a better name, please let me know.

Reply:
How about `placeholder?`?